tests/test_main.py

```diff
@@ -19,6 +19,9 @@ def setUp(self):
     def tearDown(self):
         sys.path[:] = self.old_sys_path
 
+    @unittest.skip(
```

Comment:
This test is dependent on examples folder being here, so we are skipping until we discuss testing strategy.